details.php

```diff
@@ -66,13 +66,13 @@
 }
 
 echo html_writer::alist([
-    'Name: '.$record->name,
-    'Time: '.$record->time,
-    'Points: '.$record->points,
-    'Hours: '.$record->hours,
-    'Summary: '.$record->summary,
-    'Provider: '.$record->provider,
-    'Location: '.$record->location,
+    get_string('name', 'mod_cpdlogbook').': '.$record->name,
```

Comment:
Looks good, the table headings also need sorting:

![image](https://user-images.githubusercontent.com/187449/131501678-e84d6dd4-f984-4b6f-aacd-0522895f5c79.png)

Reply:
Oops, somehow missed that and misunderstood what I was actually supposed to fix. Should actually be fixed now.

Reply:
All good, this detail page needs some proper UI love anyway